### ISSUE TITLE
Inconsstent statements in documentation

### ISSUE THREAD

**Issue by aandriasyan** [flutter-tizen/plugins#649]
According to Samsung's official developer page.
See: https://developer.samsung.com/smarttv/develop/native/flutter.html
Flutter is supported on Samsung Smart TVs starting from Tizen 6.0. 
However, the video_player_tizen plugin's documentation 
on ttps://pub.dev/packages/video_player_tizen
mentions support for devices running Tizen 5.5 and above. This presents a contradiction because if Flutter itself is only supported from Tizen 6.0 onwards, a Flutter plugin should not be operable on Tizen 5.5.

**Comment by xuelian-bai** [comment]
thanks, we will fix this

**Comment by swift-kim** [comment]
No. The doc is correct. Smart TV is not the only device that runs on the Tizen OS. My watch running Tizen 5.5 is compatible with the plugin.
